schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/fromJsonDkan.xsl

```diff
@@ -366,9 +366,9 @@
     <xsl:param name="value"></xsl:param>
 
     <!-- Regexps for parsing expected date formats -->
-    <!-- example of localized date: "jeu, 03/06/2021 - 03:00" -->
-    <xsl:variable name="regExLoc">^.*([0-9]{2})/([0-9]{2})/([0-9]{4}) - ([0-9]{2}:[0-9]{2})$</xsl:variable>
-    <!-- example of internationalized date: "2020-12-09 00:00:00" -->
+    <!-- examples of localized date: "jeu, 24/12/2020 - 03:00", "24-12-2020 03:00" -->
+    <xsl:variable name="regExLoc">^.*([0-9]{2})[-/]([0-9]{2})[-/]([0-9]{4})\s+([0-9]{2}:[0-9]{2})$</xsl:variable>
```

Comment:
Ok to accept `-` or `/`, but did you notice that the format is different as well ?

eg `"metadata_created": "11-03-2021 14:00",` is `DD-MM-YYYY` and not `YYYY-MM-DD`
It means that the comment below is deprecated cause the expression does not support this format `2020-12-09 00:00:00`
I mentioned that DKAN seems to output all kinds of date format, see https://opendata.hautsdefrance.fr/?q=api/3/action/current_package_list_with_resources which should be supported as well.

Also don't you forget the `\` before the slash ? I had to add them to test the regexp on https://regex101.com/
```
^.*([0-9]{2})[-\/]([0-9]{2})[-\/]([0-9]{4})\s+([0-9]{2}:[0-9]{2})$
```
But maybe in XSL it's ok

Thanks

Reply:
Thanks for the feedback !

The date format `2020-12-09 00:00:00` is handled by the `regExInt` one line below.

Indeed, the `\` is apparently not supported in XSL here (causing "invalid escape sequence" errors). https://regexr.com/ does not reclaim it, neither.

I've added the potential `-` separator between date and time (which I forgot) and added accepting optional seconds (which will not be used via the capturing group but stay `:00` for the sake of simplicity).

Reply:
Does it match `2020-12-09 00:00:00` ?

Reply:
This [one](https://github.com/georchestra/geonetwork/pull/231/files#diff-de68101e372db4606d7e93bd19f70ef6aae55a38ee14368eadd3c129af869658L372) does I think, doesn't it?

Reply:
yes sorry looks good !